### PR TITLE
Added an if-statement for the useless button

### PR DESCRIPTION
**Pull request by Punchkin**
Just added an if-statement for the "This button does nothing"-button so users can write their own code for the button.
This doesn't change much but it makes things for "newcomers" a lot easier.

### CODE REVIEW

**Comment by Cody-Soultz** [comment]
I should have looked at this PR before doing my code lol. Nice catch I had to find it on my own. #44

File: EvolvioColor/EvolvioColor.pde [hunk 1]
```diff
@@ -162,6 +162,8 @@ void mousePressed() {
               evoBoard.playSpeed *= 2;
             }
           }
+        } else if (buttonNum == 7) {
+          //Gets executed when the "This button does nothing"-button is clicked.
```
Comment (mbertheau) [inline EvolvioColor/EvolvioColor.pde]: It's customary to add a space after `//` :)

File: EvolvioColor/Board.pde [hunk 2]
```diff
@@ -92,7 +92,7 @@ class Board {
       fileSaveCounts[i] = 0;
       fileSaveTimes[i] = -999;
     }
-    userControl = true;
+    userControl = false;
```
Comment (mbertheau) [inline EvolvioColor/Board.pde]: You later added a commit that changes `userControl` back to `true`. Can you remove this and the following commit from the branch? Their cumulative effect is zero, so if we leave them in, they're just noise in the commit history.

Thanks!

**Comment by mbertheau** [comment]
(Just repeating this here in case it gets overlooked in the "Show outdated" section.)

You first added a commit that changes `userControl` to `false`, and then another one that changes it back to `true`. Can you remove these two commits from the branch? Their cumulative effect is zero, so if we leave them in, they're just noise in the commit history.

Thanks!

**Comment by Punchkin** [comment]
Okay. I'll do it as soon as I'm back home (currently on vacation). So...
Guess this will take some time.
I'll be back in one week :)

Am 16.10.2016 12:58 schrieb "Markus Bertheau" <notifications@github.com>:

(Just repeating this here in case it gets overlooked in the "Show outdated"
section.)

You first added a commit that changes userControl to false, and then
another one that changes it back to true. Can you remove these two commits
from the branch? Their cumulative effect is zero, so if we leave them in,
they're just noise in the commit history.

Thanks!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/evolvio/evolv.io/pull/42#issuecomment-254040126>, or mute
the thread

**Comment by Punchkin** [comment]
Seems like I'm too dumb to delete those commits. Maybe you could cancel this pull request and open a new one. 
Or teach me how to delete them (I'm a total beginner with GitHub).

**Comment by PRabahy** [comment]
@Punchkin The easiest way to fix this is going to be a reset and a force push.

`git reset ef9ca97` will reset to the commit before you added either of the commits that cancel each other out.
`git push -f` will push that change up to github and this pull request will automatically detect the change.

Also, in general its not the best idea to develop directly on master if multiple people are working on the project. If somebody else has a pull request get merged while you are still working on your change, it will cause some headaches for you. I recommend always making a new branch `git checkout -b newbranch` before you commit any changes.

**Comment by mbertheau** [comment]
Which git client are you using?

Punchkin <notifications@github.com> schrieb am Sa., 22. Okt. 2016, 22:12:

**Pull request closed without merging**